R/tbl-df.r

```diff
@@ -2,6 +2,64 @@
 #' @exportClass tbl_df
 setOldClass(c("tbl_df", "tbl", "data.frame"))
 
+#' `tbl_df` class
```

Comment:
Does this work with the LaTeX manual?

Reply:
Seems OK?

<img width="481" alt="screen shot 2018-10-22 at 3 39 47 pm" src="https://user-images.githubusercontent.com/599454/47323464-bfc54f00-d610-11e8-95b2-f8e5e9be1658.png">